fvwm/cursor.c

```diff
@@ -50,7 +55,8 @@ int num_barriers = 0;
 void create_barrier(int x1, int y1, int x2, int y2)
 {
 	if (num_barriers >= MAX_BARRIERS) {
-		fvwm_debug(__func__, "Too many barriers. Aborting.");
+		fvwm_debug(__func__, "Too many barriers. Freeing previous...");
+		destroy_all_barriers();
```

Comment:
What is your reasoning here? If they reach max barriers do you just want a reset? Since you are destroying all of the barriers first, should you remove the return bellow this so the current one they are trying to create gets created?

Reply:
You might want to document this behavior in the manual page.

Reply:
It's odd either way.

No one's going to reach this limit.